templates/central/messages.html.twig

```diff
@@ -31,35 +31,37 @@
  # ---------------------------------------------------------------------
  #}
 
+{% import 'components/alerts_macros.html.twig' as alerts %}
+
 <div class="message-area">
-   {% if messages.errors is defined and messages.errors|length > 0 %}
-      <div class="alert alert-important alert-danger d-flex" role="alert">
-         <i class="ti ti-alert-triangle fs-3x"></i>
-         <ul>
-            {% for error in messages.errors %}
-               <li>{{ error|raw }}</li>
-            {% endfor %}
-         </ul>
-      </div>
-   {% endif %}
-   {% if messages.warnings is defined and messages.warnings|length > 0 %}
-      <div class="alert alert-important alert-warning d-flex" role="alert">
-         <i class="ti ti-alert-triangle fs-3x"></i>
-         <ul>
-            {% for warning in messages.warnings %}
-               <li>{{ warning|raw }}</li>
-            {% endfor %}
-         </ul>
-      </div>
-   {% endif %}
-   {% if messages.infos is defined and messages.infos|length > 0 %}
-      <div class="alert alert-important alert-info d-flex" role="alert">
-         <i class="ti ti-info-circle fs-3x"></i>
-         <ul>
-            {% for info in messages.infos %}
-               <li>{{ info|raw }}</li>
-            {% endfor %}
-         </ul>
-      </div>
-   {% endif %}
+    {% if messages.errors is defined and messages.errors|length > 0 %}
+        {% set alert_messages %}
+            <ul>
+                {% for error in messages.errors %}
+                    <li>{{ error|raw }}</li>
+                {% endfor %}
+            </ul>
+        {% endset %}
+        {{ alerts.alert_danger(title: alert_messages, important: true) }}
```

Comment:
1. Why passing messages in the `title` param and not in the `messages` param ?

2. It seems that the output will be escaped, but the messages contains HTML code that should not be displayed. We cannot use the `|raw` filter in the alert macros if it is not explicitey required by the caller, or we will risk XSS issues. It has to be fixed.

The same remarks apply for the other calls to alets macros in the current file.